lib/homePage.dart

```diff
@@ -188,27 +194,48 @@ class HomePageView extends State<HomePage>
               child: Center(
                 child: Column(
                   children: [
-                    Transform.scale(
-                      scale: _buttonScale,
-                      child: MaterialButton(
-                        shape: RoundedRectangleBorder(
-                            borderRadius: BorderRadius.circular(30)),
-                        child: Text(
-                          buttonText,
-                          style: TextStyle(color: Colors.white, fontSize: 15),
-                        ),
-                        color: Colors.blue,
-                        onPressed: () {
-                          if (_toggled) {
-                            buttonText = "Check Heart Rate";
-                            _untoggle();
-                          } else {
-                            buttonText = "Stop";
-                            _toggle();
-                          }
-                        },
-                      ),
-                    ),
+                    Row(
+                        mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                        children: [
+                          Transform.scale(
+                            scale: _buttonScale,
+                            // child: Row(
+                            //     mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                            //     children: <Widget>[
+                            child: MaterialButton(
+                              shape: RoundedRectangleBorder(
+                                  borderRadius: BorderRadius.circular(30)),
```

Comment:
keep the border-radius to be low so that they look more like cards rather than buttons